pkgs/applications/networking/flexget/default.nix

```diff
@@ -9,23 +9,30 @@
 with pythonPackages;
 
 buildPythonPackage rec {
-  version = "1.2.337";
+  version = "2.8.17";
   name = "FlexGet-${version}";
-  disabled = isPy3k;
 
   src = fetchurl {
     url = "mirror://pypi/F/FlexGet/${name}.tar.gz";
-    sha256 = "0f7aaf0bf37860f0c5adfb0ba59ca228aa3f5c582131445623a4c3bc82d45346";
+    sha256 = "e707bcbdf6d17244bc104753274f33044a4140a8aaac91016cf4a2f9d1f4e5bf";
   };
 
+  # Unclear what it wants, but doesn't work.
   doCheck = false;
+  checkPhase = ''
+    py.test
+  '';
 
```

Comment:
It seems a significant part of the test suite is not included
https://github.com/Flexget/Flexget/tree/develop/flexget/tests
You might want to fetch from GitHub instead of PyPI.